README.md

```diff
@@ -57,18 +55,19 @@ You can easily run `tap-postgres` by itself or in a pipeline using [Meltano](htt
 ### Executing the Tap Directly
 
 ```bash
-tap-postgres --version
-tap-postgres --help
-tap-postgres --config CONFIG --discover > ./catalog.json
+meltanolabs-tap-postgres --version
+meltanolabs-tap-postgres --help
+meltanolabs-tap-postgres --config CONFIG --discover > ./catalog.json
```

Comment:
The script name in `pyproject.toml` is still `tap-postgres`, and it's independent of the package name

```suggestion
tap-postgres --version
tap-postgres --help
tap-postgres --config CONFIG --discover > ./catalog.json
```

Reply:
I am cheating a little here as https://github.com/MeltanoLabs/tap-postgres/pull/115/files has the name change. I think we need to make the name be something different for `pypi` and I only have this for the `pipx` install which I think is correct., maybe I'm still wrong?